internal/pprof/binutils/binutils.go

```diff
@@ -433,10 +433,8 @@ func (b *binrep) openELF(name string, start, limit, offset uint64, relocationSym
 	defer ef.Close()
 
 	buildID := ""
-	if f, err := os.Open(name); err == nil {
-		if id, err := elfexec.GetBuildID(f); err == nil {
-			buildID = fmt.Sprintf("%x", id)
-		}
+	if id, err := elfexec.GetBuildID(ef); err == nil {
```

Comment:
We shouldn't change the code in `internal/pprof`. This is vendored code, and all the changes will be lost when we run the script to update.

Reply:
oh, ok 👍